lib/mixpanel-node.js

```diff
@@ -151,6 +240,128 @@ var create_client = function(token, config) {
     };
 
     /**
+     * breaks array into equal-sized chunks, with the last chunk being the remainder
+     * @param {Array} arr
+     * @param {number} size
+     * @returns {Array}
+     */
+    var chunk = function(arr, size) {
+        var chunks = [],
+            i = 0,
+            total = arr.length;
+
+        while (i < total) {
+            chunks.push(arr.slice(i, i += size));
+        }
+        return chunks;
+    };
+
+    /**
+     * helper to wait for all callbacks to complete; similar to `Promise.all`
+     * @param {Array} requests
+     * @param {Function} handler
+     * @param {Function} callback
+     */
+    var asyncAll = function(requests, handler, callback) {
```

Comment:
Nice. Any day now we'll get around to dropping support officially for any node.js <v4.2 and can just use Promises natively. In the meanwhile, would you mind adding some basic tests for this helper? Doesn't have to be comprehensive, just enough to give some confidence in its behavior.